.gitignore

```diff
@@ -13,3 +13,4 @@ guides/common/default.css
 /web/crash.log
 /web/.byebug_history
 /web/output/
+/vendor
```

Comment:
Closed a PR yesterday when a contributor committed this, making sure this does not stand in the way.